.github/workflows/mac.yml

```diff
@@ -56,6 +56,14 @@ jobs:
     name: Mac Build
     runs-on: macos-13
     steps:
+      # XProtect can cause random failures if it decides that the DMG we create
+      # during the packaging phase is malware.
+      # TODO(mrobinson): Is there a way we can do things in a less suspicious way so
+      # we don't have to kill this service?
+      - name: Kill XProtectBehaviorService
+        run: |
+          echo Killing XProtect.; sudo pkill -9 XProtect >/dev/null || true;
+          echo Waiting for XProtect to be killed...; while pgrep XProtect; do sleep 1; done;
```

Comment:
Could we just skip this and hope that Xprotect will be killed during next steps?

Reply:
Sure. I've done that in the latest version of the branch.